IS0402Test.py

```diff
@@ -2223,3 +2098,8 @@ def post_resource(self, test, type, data, code=None):
                   and not r.headers["Location"].startswith(self.protocol + "://")):
                 raise NMOSTestException(test.FAIL("Registration API response Location header is invalid for the "
                                                   "current protocol: Location: {}".format(r.headers["Location"])))
+
+    def check_api_v1_x(self, test):
```

Comment:
I don't have a problem with including this for now (as it matches existing behaviour), but I wonder if we need a more general global test suite check when submitting that the major version isn't too high? Or perhaps this is overkill in general as we'd need to update all of these tests if before introducing a v2.0 option to the dropdown anyway?

Reply:
Yes, I wasn't quite sure about the right way to go with this. It seemed beneficial while refactoring this suite to make this check a precondition rather than `if`...`else`. As you say, v2.0 seems a long way off, so perhaps it's not worth too much thought right now...